src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/data-broker-profiles/welcome-to-premium/WelcomeToPremiumView.tsx

```diff
@@ -32,6 +35,18 @@ export function WelcomeToPremiumView(props: Props) {
   );
   const dataPointReduction = getDataPointReduction(summary);
 
+  // MNTOR-2594 - log any users that are on welcome-to-premium page but not subscribed.
+  getServerSession(authOptions)
+    .then((session) => {
+      if (!session?.user.fxa?.subscriptions.includes("monitor")) {
+        logger.error("user_not_subscribed", {
+          page: "welcome-to-premium",
+          subscriber_id: session?.user.subscriber?.id,
```

Comment:
Don't we want to avoid logging this? Or is this just for foxfooding? (If so, maybe we could add a condition to limit this to foxfooders, or to deactivate after a certain date?)

Reply:
I'll go ahead and remove it, the most important thing is that we count how many times and when this occurs. I'll send it to Sentry as well, since it's something we'll want to show up easily in triage.